path_2023.py

```diff
@@ -27,6 +30,11 @@ def run() -> int:
     if not result:
         print("ERROR: load_waypoint_name_to_coordinates_map")
         return -1
+    
+    result = waypoints_dict_to_kml.waypoints_dict_to_kml(waypoint_name_to_coordinates, KML_FILE_NAME, KML_FILE_PATH)
```

Comment:
Call `waypoints_dict_to_list()` (or whatever you name it) followed by `waypoints_to_kml()` .